nCompiler/R/compile_labelAbstractTypes.R

```diff
@@ -170,13 +176,20 @@ inLabelAbstractTypesEnv(
                              AssignAfterRecursing(code, symTab, auxEnv,
                                                   handlingInfo))
             }
+            if (logging) {
```

Comment:
Ditto: I was thinking that this could be done generically in`compile_labelAbstractTypes` after returning from a handler.

Reply:
Updated in d76965e.